src/components/List/_List.scss

```diff
@@ -10,6 +10,52 @@
 .list {
   margin: unset; // browser automatically applies margin
 
+  &__definition-list {
+    border-bottom: 1px solid var(--ui-gray-light);
+
+    h3 {
+      border-top: 3px solid var(--section-research-primary);
+      padding-top: var(--space-xs);
+    }
+
+    dl {
+      display: grid;
+      grid-template-columns: 100%;
+      grid-template-rows: 1fr;
+      margin-bottom: 0;
+
+      dt {
+        border-top: 1px solid var(--ui-gray-light);
+        font-size: var(--font-size-0);
+        font-weight: bold;
+        padding-bottom: 0;
+        padding-right: var(--space-m);
+        padding-top: var(--space-xs);
+
+        @include breakpoint($breakpoint-medium) {
+          padding-bottom: var(--space-s);
+        }
+      }
+
+      dd {
+        border-top: none;
+        font-size: var(--font-size-0);
+        margin: 0;
+        padding-bottom: var(--space-s);
+        padding-top: var(--space-xs);
+
+        @include breakpoint($breakpoint-medium) {
+          border-top: 1px solid var(--ui-gray-light);
+        }
+      }
+
+      @include breakpoint($breakpoint-medium) {
+        grid-template-columns: max(225px) 1fr;
+        margin-bottom: var(--space-s);
```

Comment:
Looking at the Figma, it would be good to include something to set a min height for the rows.  I believe `grid-auto-rows` could be used for this.  Adding this, however, may also impact spacing that has already been set in other margin/padding vlaues.

Reply:
Not going to do this now.  Will address later, if need be.